improver/calibration/ensemble_calibration.py

```diff
@@ -1127,25 +1425,35 @@ def _calculate_location_parameter_from_realizations(self):
                 as the predictor.
         """
         forecast_predictor = self.current_forecast
-
         # Calculate location parameter = a + b1*X1 .... + bn*Xn, where X is the
         # ensemble realizations. The number of b and X terms depends upon the
         # number of ensemble realizations. In this case, b = beta^2.
-        beta_values = np.array([], dtype=np.float32)
-        beta_values = self.coefficients_cubelist.extract_strict(
-            "emos_coefficient_beta"
-        ).data.copy()
-        a_and_b = np.append(
-            self.coefficients_cubelist.extract_strict("emos_coefficient_alpha").data,
-            beta_values ** 2,
+        beta_cube = self.coefficients_cubelist.extract_strict("emos_coefficient_beta")
+        beta_values = np.atleast_2d(beta_cube.data ** 2)
```

Comment:
Having spotted this one and asked you to change it, I've now done a search through the code and found 9 other instances of squaring data using `** 2` rather than multiplying data by itself. As the slow-down can be up to x3.5 (StackOverflow), it is probably worth changing all of them. A shame really as `** 2` looks much nicer in a programme.

Reply:
I've updated this as suggested.